common/djangoapps/student/views.py

```diff
@@ -2244,6 +2244,13 @@ def password_reset_confirm_wrapper(request, uidb36=None, token=None):
             request, uidb64=uidb64, token=token, extra_context=platform_name
         )
 
+        # If password reset was unsuccessful a template response is returned (status_code 200).
+        # Check if form is invalid then show an error to the user.
+        # Note if password reset was successful we get response redirect (status_code 302).
+        if response.status_code == 200 and not response.context_data['form'].is_valid():
+            response.context_data['err_msg'] = 'Password reset form did not validate, please try again'
+            return response
+
         # get the updated user
         updated_user = User.objects.get(id=uid_int)
```

Comment:
oh, nvm.  I misread the code.  I now see that there's a `return` statement (on L2252) before the rest of this clause.   

(Side note: per "clean code" tips, it's generally better to implement your function so there's only one return statement in the function.  Having a single exit flow makes the code easier to debug and to read.  I'm not requiring that we make this change now, but something to keep in mind.)